### ISSUE TITLE
Website need to be bilingual

### ISSUE THREAD

**Issue by vivrass** [montrealrb/Montreal.rb#4]
French + English

**Comment by nicholasjhenry** [comment]
Let's add this as a requirement to CONTRIBUTING.md and then close this issue.

**Comment by sophiedeziel** [comment]
I agree for the long term, but let's not worry about i18n for the workshop if some teams are not familiar with this.

**Comment by nicholasjhenry** [comment]
ok, so you're suggesting we won’t have translated models for the workshop?

**Comment by sophiedeziel** [comment]
Actually, this is something worth thinking about. I honestly don't know what's the overhead of translations for beginners. Also, as we discussed Wednesday, the goal is to have the set of features we use in the actual wordpress website. Translation is not part of it. 

But, internationalizing from the start has so many benefits on the long term and is definitely something i'd love to have.

Your idea of having internationalization as a requirement for any contribution is really good. My question is about the short term and how we will "enforce" that at the workshop where we don't have a clear idea of the skill sets of the participants :)

**Comment by nicholasjhenry** [comment]
the other thing to consider is that  the existing models are translated, but I share your concern the barrier to entry for beginners.

**Comment by mathieugagne** [comment]
I made a comment on Slack about translation not being a requirement for a minimum viable product. We do everything in English at the moment. It also seems that the approach to implement it wasn't as straight forward:
- internationalized db columns
- globalize
- pure I18n yaml translations
- custom translation system

I personally would consider this a feature and not a requirement. Would focus on it after the initial launch so we get something out faster. Our approach might change based on what our MVP looks like.

My 2 cents.

**Comment by sophiedeziel** [comment]
@mathieugagne did a great job summarizing our discussion on slack. Also, after spending some time thinking about it, I agree with his 2 cents :wink:. Finally, Mathieu is leaving cents everywhere, we will have a dollar pretty soon!

**Comment by nicholasjhenry** [comment]
Since we have translated models already, why don't we do keep doing translated models (i.e. Globalize), but ignore all other aspects of I18n?
